recommenders/__init__.py

```diff
@@ -24,10 +24,6 @@
 
 
 
-
-movies_df = pd.read_csv("movies.csv")
-ratings_df = pd.read_csv("ratings.csv")
-

```

Comment:
The 'default_poster_url' variable is not used in the code. Consider removing or utilizing it.